frontend/containers/AuctionsFakeData.vue

```diff
@@ -120,6 +160,8 @@ export default Vue.extend({
                     this.selectedAuction.isFinished = true;
                     this.selectedAuction.endDate = new Date();
                     this.selectedAuction.transactionAddress = faker.finance.ethereumAddress();
+                    this.walletVatDai = this.walletVatDai.minus(this.transactionAmountDai!!);
+                    this.transactionAmountDai = new BigNumber(NaN);
```

Comment:
Why does calling `authorizeWallet` sets `transactionAmountDai` to NaN?

Reply:
Uhm.. good question